docs/resources/elastic_ip.md

```diff
@@ -49,9 +50,9 @@ directory for complete configuration examples.
 [zone]: https://www.exoscale.com/datacenters/
 
 * `zone` - (Required) The Exoscale [Zone][zone] name.
-
```

Comment:
Please keep the separation between _(Required)_ and optional arguments blocks

Reply:
Fixed, thanks.